eipw-lint/src/lints/markdown/relative_links.rs

```diff
@@ -23,15 +23,19 @@ pub struct RelativeLinks;
 
 impl Lint for RelativeLinks {
     fn lint<'a, 'b>(&self, slug: &'a str, ctx: &Context<'a, 'b>) -> Result<(), Error> {
-        let re = Regex::new("(^/)|(://)").unwrap();
+        let re = Regex::new(r"(^/)|(://)|(^www)|^(\w)+\.(\w)+").unwrap();
```

Comment:
Would this treat `[hello](www)` as a relative link?

Reply:
No, that is not allowed, should it work? I can specify it as `www` in front of domain.

Reply:
Yeah, I think `[hello](www)` will turn into [hello](www). On this particular PR that expands to `https://github.com/ethereum/eipw/pull/www`.